app/policies/direct_assessment_policy.rb

```diff
@@ -0,0 +1,13 @@
+class DirectAssessmentPolicy < ApplicationPolicy
+  def create?
+    GenericPolicy.new(user, record).create_assessments?
```

Comment:
At the point of creation you *will* know enough information to determine the applicable department, correct? So we'd want to go based on that, right? What about manually using the generic policy in the new action itself and then using the more specific `DirectAsessmentPolicy` on create?